sdv/single_table/ctgan.py

```diff
@@ -154,7 +154,7 @@ def _estimate_num_columns(self, data):
                 num_generated_columns[column] = 11
 
             elif sdtypes[column] in {'categorical', 'boolean'}:
-                if transformers[column] is None:
+                if transformers.get(column) is None:
```

Comment:
nit: Can we do `transformers.get(column, None)` so it's explicit?

Reply:
`python` always returns `None`.